README.md

```diff
@@ -34,13 +34,21 @@ JavaScript communicates with the game via events and APIs exposed by the game, a
 
 ## Setup
 
-TODO
-(reference https://github.com/panorama-languages-support/vscode-panorama-css)
+The easiest way to develop for Momentum Mod Panorama is to mount this repo somewhere in your `momentum/custom/` folder, for example
+
+```
+cd <Your Steam Momentum Mod or Momentum Mod Playtest install location>\momentum\custom\
+mkdir PanoramaDev
+cd PanoramaDev
+git clone https://github.com/momentum-mod/panorama.git
```

Comment:
People can clone but they won't be able to push directly to a clone of this repo. I'd suggest maybe making this step something like

```suggestion
(create a clone of your fork here)
```

The end result should be `custom/PanoramaDev/panorama`, but their repo.

Reply:
This okay now?